README.md

```diff
@@ -129,6 +129,44 @@ The script will load the ZKPI data for the specified block, convert it to the
 format required by Keth, run the proof generation process, and save proof
 artifacts to the output directory.
 
+### Running State Transition Function from ZKPI
+
+The `zkpi_to_stf.py` script allows you to execute and verify Ethereum block
+state transitions in a stateless manner using Python. This tool processes
+ZK-Prover Input (ZKPI) files generated by ZK-PIG (ZK-Prover Input Generator) to:
+
+1. Load the pre-state MPT (Merkle Patricia Trie) with all necessary nodes and
+   code
+2. Execute the block's transactions against this state
+3. Verify that the resulting state root matches the expected state root in the
+   block header
+4. If verification fails, identify and report discrepancies in account and
+   storage values
+
+This provides a way to validate Ethereum state transitions without maintaining a
+full node state, using only the specific trie nodes required for the block's
+execution.
+
+Usage:
+
+```bash
+python cairo/scripts/zkpi_to_stf.py <block_number> [--path <path_to_input_files>]
```

Comment:
I still don't understand why we want 2 args and not only a zkpi file path

Reply:
it's mainly preference, i'm fine with adding just 1 arg

Reply:
but I prefer two args -> it's simpler to call

Reply:
yagni